src/backend/sgx/mod.rs

```diff
@@ -198,6 +203,21 @@ impl super::Thread for Thread {
                     },
 
                     SYS_ERESUME => Entry::Resume,
+                    SYS_GETATT => {
+                        let result = unsafe {
+                            get_attestation(
+                                self.block.msg.req.arg[0].into(),
+                                self.block.msg.req.arg[1].into(),
+                                self.block.msg.req.arg[2].into(),
+                                self.block.msg.req.arg[3].into(),
+                            )?
+                        };
+
+                        // TODO: Probably a better rdx to return here
```

Comment:
doesn't matter

Reply:
@haraldh I can remove the comment - is there anything else to change in the PR?

Reply:
not besides `internal/shim-sgx/src/handler.rs` ... and btw, you should change the signature of the `get_attestation` method in the trait also to match the arguments used in the syscall definition (like all the other syscalls). Then you don't have to use `self.aex.gpr.*`

Reply:
@haraldh I updated the function signatures for `get_attestation()` - is this all you meant by "not besides `handler.rs`"?